src/css/lesson.css

```diff
@@ -0,0 +1,198 @@
+:root {
+    --black: #121417;
+    --green: #4DC274;
+    --light-green: #bee8cc;
+    --white: #fff;
+    --font-family: "Manrope", sans-serif;
+}
+
+body {
+    font-family: var(--font-family);
+    margin: 0;
+    padding: 0;
+    background-color: var(--white);
+}
+
+/* Секція уроків */
+.lessons {
+    text-align: center;
+    padding: 50px 0;
+}
+
+.lessons-container {
+    max-width: 1280px;
+    margin: 0 auto;
+    padding: 0 20px;
+}
+
+.lessons-title {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 120%;
+    letter-spacing: -0.02em;
+    margin-bottom: 20px;
+    position: relative;
+    display: inline-block;
+}
+
+.underline-lessons {
+    display: block;
+    position: absolute;
+    left: 60%;
+    bottom: -1px;
+    transform: translateX(-38%);
+    width: 63%;
+    height: 20px;
+    background-color: var(--light-green);
+    border-radius: 8px;
+    z-index: -1;
+}
+
+.lessons-text {
+    font-size: 18px;
+    line-height: 167%;
+    text-align: center;
+    color: rgba(18, 20, 23, 0.5);
+    margin-top: 16px;
+    margin-bottom: 64px;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 665px;
+}
+
+/* Контейнер для планів */
```

Comment:
pls do according to design

for desktop show all three plans - it ok
for taablet you shoul have 2 columns (2 plans in first row + 1 on onother)
for mobile you should have all plans in 1 column